frontend/src/app/providers/AuthProvider.tsx

```
@@ -1,130 +1,155 @@
+import { REFRESH_TOKEN_MUTATION } from '@/graphql/mutations/auth';
+import { LocalStore } from '@/lib/storage';
 
-interface AuthProviderProps {
-  children: React.ReactNode;
+interface AuthContextValue {
+  isAuthorized: boolean;
+  isChecking: boolean;
+  token: string | null;
+  login: (accessToken: string, refreshToken: string) => void;
+  logout: () => void;
+  refreshAccessToken: () => Promise<string | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)